packages/bot/src/components/contract-card.jsx

```diff
@@ -45,7 +42,7 @@ const ContractCard = ({
                 { is_contract_completed && <ContractResultOverlay profit={contract.profit} /> }
                 <div className='db-contract-card__underlying'>
                     <div className='db-contract-card__underlying-name'>
-                        <UnderlyingIcon market={contract.underlying} />
+                        <Icon icon={`IcUnderlying${contract.underlying}`} width={32} height={24} />
```

Comment:
Some of the icons have different dimensions e.g. 32x32 or 28x22, I think setting this to 32x32 will suffice.

```suggestion
                        <Icon icon={`IcUnderlying${contract.underlying}`} size={32} />
```